### DISCUSSION TITLE
The Share button of Android APP

### DISCUSSION THREAD

**Discussion by AlliotTech** [immich-app/immich#4469] [General]
Hi,
I have a question about the Share function:  

On immich website, the share button means to create a share link, while in the Android APP it means to download the source file and share it via the system's share feature.    

And I don't find a place to create a share link in the app.
 
It's very confusing.  Wouldn't it be more appropriate to standardize the logic?

**Comment by alextran1502** [comment]
Hello, it still needs to be implemented in the app for shared link.

**Comment by bo0tzz** [comment]
See #1863
